docs/build-parachains.md

```diff
@@ -0,0 +1,174 @@
+around your project and convincing validators to participate in your network security are reduced.
+Polkadot has strong security, and decentralized application projects wishing to benefit from this
+security would want to become a parachain to share in that pooled security. For more information
+on the mechanic of leasing a parachain slot through a candle auction see [here](learn-auction).
+
+Any decentralized application or chain that wants to enable trustless messaging to other parachains
+already connected to Polkadot would want to become a parachain. Interoperability between sovereign
+chains involves certain constraints and complex protocols to enable across a wide breadth of chains.
+With Polkadot, you will get this feature out of the box if your build your application as a
+parachain. The [XCMP protocol](learn-crosschain) allows any parachains to communicate by
+passing messages between them. Furthermore, as bridges to other chains are connected (such as those
+to Bitcoin or Ethereum) Polkadot's parachains will be able to communicate with these as well!
+
+## What is a Parachain Development Kit (PDK)?
+
+A PDK is a set of tools that allows developers to easily create a parachain. In
+practice this means that the PDK will consist of a couple key components:
+
+- _State transition function_ - a way for your application to move from one state to another state.
+- _Collator node_ - a type of peer-to-peer node in the Polkadot network with certain
+  responsibilities in regard to parachains.
+
+The state transition function (STF) can be any abstract way for an application to go from one state
+to another state. The only constraint that Polkadot places on this STF is that it must be easily
+verifiable -- usually though what we call a _witness_ or _proof_. It must be so because the Relay
+Chain validators will need to check that each state it receives from the collator node is correct
+without actually running through the entire computation. Some examples of these proofs include the
+Proof-of-Validity blocks or zk-SNARKs, which require less computational resources to verify than
+they do to generate. The verification asymmetry in proof generation of the STF is one of the
+integral insights that allows Polkadot to scale while keeping high security guarantees.
+
+A collator node is one of the types of network maintainers in the Polkadot protocol. They are
+responsible for **keeping availability** of the state of the parachain and the new states returned
+from iteration of the state transition function. They must remain online in order to keep track of
+the state and also of the XCMP messages that it will route between itself and other parachains.
+Collator nodes are responsible for passing the succinct proofs to the relay chain's validators, and
+tracking the latest blocks from the relay chain. In essence, a collator node also acts as a light
+client for the relay chain. For more on collator nodes see [here](learn-collator).
+
```

Comment:
Since there is discussion of how the collator's role, it would be good to have an example of steps to put these concepts in perspective; taking the case for one parachain so to speak 

E.g. start of a new block candidate - collator aggregates the transactions -> collator signs parachain block candidate and produces state transition proofs -> passes the candidate block and state transition proofs to validators -> validators verify transactions within the candidate -> shares with relay chain -> parachain block candidates collected together and relay chain block candidate produced